### ISSUE TITLE
Two tooltips are displayed for listview icons to log call, schedule meeting, create task, compose email

### ISSUE THREAD

**Issue by gmcodentech** [salesagility/SuiteCRM-Core#540]
### Issue

In SuiteCRM 8, I found when I place the mouse pointer on the phone icon in opportunity module list view, it shows ‘phone icon’ and when I place the mouse pointer just below the phone icon, it shows ‘log a call’.

This is present for other icons for schedule meeting, create task and compose email and in the list of other modules as well.

![24e4efa260a582fe44fe4e2d8b4e053ae1db5184](https://github.com/user-attachments/assets/72b3576a-de84-42ba-95db-3d37646afda7)

![c6cd15c9188c0a0162a5a9b4f62c3aa503ed7ec9](https://github.com/user-attachments/assets/fbab0d4f-1250-49cd-b74c-b5f09105f43a)

I am not sure if this is a small issue, or this is what it should behave.
Thanks

### Possible Fix

Remove 'phone icon' and keep 'Log Call'. Similarly remove one for others 

### Steps to Reproduce the Issue

```bash
1. Login into SuiteCRM 8
2. Open Accounts Or Contacts Or Opportunities Module List View
3. On First Row, we can icons for phone, email, tasks and schedule a meeting
4. Place mouse pointer on icon to see the first tooltip
5. Place mouse pointer just below the same icon to see the second tooltip
```

### Version

8.6, 8.7 Beta

### What browser are you currently using?

Chrome

### Environment Information

MySQL

### Operating System and Version

Windows

**Comment by pgorod** [comment]
There are more tooltips describing the icon, in more places, such as "Calendar icon", I guess a more generic fix is in order.

**Comment by shubham-pawar** [comment]
@pgorod you are correct. I checked them in the v8.8 too. Someone need to update those onHover labels for all icons.
